### DISCUSSION TITLE
🥂 List of changes and suggestions.

### DISCUSSION THREAD

**Discussion by Stainded** [diamond-rip/Eden#210] [Ideas]
_Please note that all these suggestions are my predictions of what can be added to the plugin, they are extremely important things that I may be able to suggest in which versions to add._
**I AM NOT A DEVELOPER WHO HAS ANY RELATION WITH THIS PLUGIN, I AM JUST A USER WHO WANTS TO IMPROVE THIS PLUGIN BECAUSE IT IS SO SUCCESSFUL.**

* Suggestions for the (**/settings**) system; [Which could be added perhaps in **Eden-1.1.7**]
  - Add a message system where it warns that such a preference has been changed to "ENABLED or DISABLED" instead of just a message stating that such a setting has been changed.
  - Add a PLACEHOLDER where you put glass in the empty slots ("**STAINED_GLASS_PANE** and **DURABILITY: custom_value**"), set the size of the settings menu and the position of each preference selector.
  --> FILLED_EMPTY_SLOT_EXAMPLE: (ITEM: STAINED_GLASS_PANE, DURABILITY: 15)
![image](https://user-images.githubusercontent.com/128519059/227111399-9a937bba-c799-4142-912b-3869ab6119a0.png)

  -  Remove "**MATCH-SCOREBOARD**" and replace or add remove or activate "**SCOREBOARD-GLOBAL**", add system add or remove "**BODY-ANIMATION**" when deleting player from some match event.

* Suggestion for the (**PARTY**) system in the plugin; [Which could be added perhaps in **Eden-1.1.6 or 1.1.9**]
  -  Add a command that leads directly to the party game modes in either (**/PARTY SPLIT BATTLEKIT**) or (**/PARTY FFA BATTLEKIT**)
  - Add queuing system for (**PARTY 2V2**, **RANKED-PARTY** or **UNRANKED-PARTY)**)

* Suggestion for the (**SPAWN**) status of the player [Which could be added perhaps in **Eden-1.2.0 or 1.1.9**]
  - Add "QUEUE-MENU" and create follow-item to (**SPECTATOR-MODE**)

* Suggestion for the (**HOST-EVENT**) menu or command/system;
  - Add custom commands, example: (**/HOST SUMO [1V1/2V2] LIMIT-PLAYERS: [60]**)
  - Update event menu adding custom PLACEHOLDER putting glass in the empty slots ("**STAINED_GLASS_PANE** and **DURABILITY: custom_value**")
  
 * Suggestion for the new (POST-MATCH) message: [Which could be added perhaps in **Eden-1.1.6 or 1.1.7**]
   -> Image example
![image](https://user-images.githubusercontent.com/128519059/227109151-5bea8f15-6a81-4470-aa71-a2137f261747.png)

* Suggestion for the messages (FOLLOW-SILENT-MODES):  [Which could be added perhaps in **Eden-1.2.0**]
  - Examples of how the message system for the plugin might work!
  - Placeholders: {silent} (SCOREBOARD) = "&fSilent &aEnabled"
![image](https://user-images.githubusercontent.com/128519059/227109972-c2857471-3f50-4cca-9fc9-526d58f3450b.png)

* Suggestion for the hotbaritems (FOLLOW-ITEMS/FOLLOW-PLACEHOLDERS):
  - Placeholders: {following} (SCOREBOARD) = "&fFollowing: &b{0}"
  - "%eden_player_following%" = "&fFollowing: &b{0}" ### {0} ->Player you are following
  - "%eden_player_silent_status%" = "&fSilent &aEnabled" ### TRUE or FALSE
![image](https://user-images.githubusercontent.com/128519059/227114305-ae3568eb-273e-4c9e-9aea-10403ca7dd70.png)

@GoodestEnglish, These are my suggestions for now.. ;D

**Comment by GoodestEnglish** [comment]
Interesting, most of the features you've listed will be slowly add into Eden.

**Comment by TehJays** [comment]
the first feature is pretty good
